sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/logs/_exporter.py

```diff
@@ -152,6 +151,9 @@ def _convert_log_to_envelope(log_data: LogData) -> TelemetryItem:
             exc_type = "Exception"
         if not exc_message:
             exc_message = "Exception"
+        # Log body takes priority for message
+        if log_record.body:
+            exc_message = str(log_record.body)
```

Comment:
This would be better. This way log record body is prioritized but we are not overwriting a variable multiple times and the priority is clearer
```python
if log_record.body:
     message = str(log_record.body)
elif exc_message:
    message = exc_message
else:
    message = "Exception"
...
exc_details = TelemetryExceptionDetails(
            type_name=str(exc_type)[:1024],
            message=str(message)[:32768], #changed from exc_message
            has_full_stack=has_full_stack,
            stack=str(stack_trace)[:32768],
        )
```